### PR TITLE
Update Cohort Information

### PR DESCRIPTION
**Pull request by kellymorin**
## Changes Made

1. Added all images, bios, links, etc to cohort.json. Site now displays all info for all classmates

1. Added 'hired' status in json file that will display a badge as classmates accept job offers

## Steps To Test

1. `git pull origin master`
1.  `git fetch --all`
1.  `git checkout update-images`
1.  start http server
1.  start json server 1: data/cohort.json
1. start json server 2: data/techs.json
1. Check all links, modals and info

### CODE REVIEW

**Comment by kellymorin** [comment]
@scivarolo could you specifically look at the resolution of the group photos? Are those high-res enough?

**Comment by jasehackman** [comment]
With the photos, I have a 4k screen so things might look more blurry on mine than it does on others. The individual photos look great but the group photos look blurry. The favicon also looks blurry on my screen.

File: data/cohort.json [hunk 1]
```diff
@@ -4,339 +4,309 @@
...
     },
     {
-      "id": 9,
+      "id": 10,
       "firstName": "Jase",
       "lastName": "Hackman",
-      "reelThemIn": "",
-      "bio": "",
+      "reelThemIn": "Jase “The Hack Man” Hackman",
+      "bio": "I have always been a strange mix of creative and analytical, but have struggled to find where that duality can coexist. As a musician I am been able to explore my creativity, and in my business ventures I have been able to poor over details and strategy, but I have been unable to marry the two. As a software developer, I am able to use my whole self to solve problems. I have the opportunity to come up with a creative solution, while at the same time, dive deep into the details in order to bring that solution to life.",
```
Comment (jasehackman) [inline data/cohort.json]: Well... I found a spelling error in my own bio.... and turns out it is also in my resume... darn. 

Please change poor to pore

**Review by jasehackman — COMMENTED** [review]
I only saw my own spelling error and the links issue. Everything else looks great.

**Comment by CurtainUp** [comment]
Thanks Section: Thank you _Kristen_ and Ashley! 
Should be **Kristin**

Current favicon is C# icon. Update to Python or 28?

Header "Cohort 28 | Demo Day March 22nd" still has clickable # hyperlink that doesn't do anything.

Class photos look lo-res.

General Design Suggestions:
- Can the text in the modals be equal to the width of the icons? Right now the text is a bit overwhelming and the flow from top to bottom gets wider from photo to icons to text.
- Unify formatting for quotes. " - Quoter" 

Personal Site Link Breakages:
Alfonso: Website going to http://localhost:8080/alfonso-miranda.com 404 Error
Andrew: Website going to http://localhost:8080/www.andrew-herring.com 404 Error
Brad: https://www.braddavistech.com/ ERR_CONNECTION_REFUSED
Bryan: http://localhost:8080/www.bryannilsen.com 404 Error
Jessica: http://localhost:8080/www.jessica-barnett.com 404 Error
John: http://localhost:8080/www.johnnathanwood.com 404 Error
Kelly: http://localhost:8080/kellymorin.tech 404 Error
Lesley: http://localhost:8080/www.lesleyaboyd.com 404 Error
Nolan: http://localhost:8080/www.nolanlittle.com 404 Error
Ousama: http://localhost:8080/ousama.me 404 Error
Rachel: http://localhost:8080/www.rachdaniel.com 404 Error
Sam: http://localhost:8080/www.samuelwebber.com 404 Error

All tech links work. 
Tested all modals and navbar scrolls. Each e-mail address is unique to the person attached.

File: data/cohort.json (hunk 1 above)
Comment (scivarolo) [reply data/cohort.json]: fixed!

**Comment by jasehackman** [comment]
Should we move the hired people down to the bottom? Because it is alphabetical the both appear at the top. Thoughts?

**Comment by scivarolo** [comment]
Sites that still don't work after adding http:// to the urls
- Alfonso
- Ousama
- Rachel

assuming they haven't launched yet.

**Comment by kellymorin** [comment]
Double check the following items:

- [x] Unify formatting for quotes

- [ ] Follow up with alfonso, link still not working

- [x] Follow up with ousama, link still not working

- [x] Follow up with rachel, link still not working

**Comment by kellymorin** [comment]
> Double check the following items:
> 
> * [x]  Unify formatting for quotes
> * [ ]  Follow up with alfonso, link still not working
> * [x]  Follow up with ousama, link still not working
> * [x]  Follow up with rachel, link still not working

Rachel will launch tonight, Ousama is undecided about launching, so his link has been removed until he is ready to link to that site

**Comment by jasehackman** [comment]
👍

**Comment by CurtainUp** [comment]
🦆

**Pull request merged**